spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/multiple/MultipleHopsIntegrationTests.java

```diff
@@ -109,11 +109,9 @@ public void should_prepare_spans_for_export() throws Exception {
 	public void should_propagate_the_baggage() throws Exception {
 		//tag::baggage[]
 		Span initialSpan = this.tracer.nextSpan().name("span").start();
-		try (Tracer.SpanInScope ws = this.tracer.withSpanInScope(initialSpan)) {
-			ExtraFieldPropagation.set("foo", "bar");
-			ExtraFieldPropagation.set("UPPER_CASE", "someValue");
-			//end::baggage[]
-		}
+		ExtraFieldPropagation.set(initialSpan.context(), "foo", "bar");
+		ExtraFieldPropagation.set(initialSpan.context(),"UPPER_CASE", "someValue");
+		//end::baggage[]
 
 		try (Tracer.SpanInScope ws = this.tracer.withSpanInScope(initialSpan)) {
 			//tag::baggage_tag[]
```

Comment:
if you look one line below here, ExtraFieldPropagation.get uses the initialSpan.context